quickwit/quickwit-control-plane/src/indexing_scheduler/scheduling/README.md

```diff
@@ -0,0 +1,105 @@
+# Scheduling logic
+
+Quickwit needs to assign indexing tasks to a set of indexers nodes.
+We call the result of this decision the indexing physical plan.
+
+This needs to be done under the constraints of:
+- not exceeding the maximum load of each node. (O)
+
+We also want to observe some interesting properties such as:
+- (A) we want to avoid moving indexing tasks from one indexer to another one needlessly.
+- (B) we want a source to be spread amongst as few nodes as possible
+- (C) we prefer to respect some margin on the capacity of all nodes.
+- (D) when we are working with the Push API source, we prefer to colocate indexers on
```

Comment:
Is "Push API source" = "Ingest API source"?

Reply:
(I think yes but we probably want to use the same naming everywhere)

Reply:
good point!